spec/inferno/dsl/fhir_client_spec.rb

```diff
@@ -75,23 +75,78 @@ def test_session_id
     end
   end
 
+  describe '#body_to_path' do
+    p1 = FHIR::Parameters::Parameter.new
```

Comment:
These should not be defined directly in a `describe` block. You should look at `let` for defining these sorts of things.

Also, variables need to have descriptive names, not things like `b` or `p1`, with abbreviations only in exceptional cases.